componentLibraries/defaultLibrary/Hydraulic/Volumes&Lines/HydraulicAckumulator.hpp

```diff
@@ -33,14 +33,15 @@
 //!
 //! @file HydraulicAckumulator.hpp
 //! @author Petter Krus <petter.krus@liu.se>
-//! @date Thu 11 Dec 2014 11:47:19
+//  co-author/auditor **Not yet audited by a second person**
+//! @date Tue 27 Aug 2019 15:09:03
 //! @brief This is piston with an inertia load
 //! @ingroup HydraulicComponents
 //!
 //==This code has been autogenerated using Compgen==
-//from 
-/*{, C:, HopsanTrunk, componentLibraries, defaultLibrary, Hydraulic, \
-Volumes&Lines}/HydraulicAckumulator.nb*/
+//from
+/*{, C:, Users, petkr14, Dropbox, AircraftDesign, AeroComponents, \
```

Comment:
I find this troublesome, that the code is generated from a source not available / synced with this code repo.
How many other components are there that we do not have updates for ?